bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/BinaryPrefix.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.library.unit;
+
+import java.math.BigInteger;
+
+import javax.measure.Quantity;
+import javax.measure.Unit;
+import javax.measure.UnitConverter;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import tec.uom.lib.common.function.SymbolSupplier;
+import tec.uom.lib.common.function.UnitConverterSupplier;
+import tec.uom.se.function.RationalConverter;
+
+/**
+ * The binary prefixes used to derive units by specific powers of 2.
+ *
+ * @author Gaël L'hopital - initial contribution and API
```

Comment:
Change the description to "Initial contribution". Please check the rest of the new classes too - especially for the uppercase "I".